src/main/java/gov/cms/madie/measure/utilities/qicore411/PackagingUtilityImpl.java

```diff
@@ -54,92 +55,53 @@ public byte[] getZipBundle(Export export, String exportFileName) throws Internal
-      entry.setSize(xmlBytes.length);
-      zos.putNextEntry(entry);
-      zos.write(xmlBytes);
-      zos.closeEntry();
+      entries.put(exportFileName + ".xml", xmlBytes);
+
       // add Library Cql Files to Export
       List<CqlLibrary> cqlLibraries = getCQLForLibraries(bundle);
       for (CqlLibrary library : cqlLibraries) {
         String filePath =
             CQL_DIRECTORY + library.getCqlLibraryName() + "-" + library.getVersion() + ".cql";
         String data = library.getCql();
-        entry = new ZipEntry(filePath);
-        entry.setSize(data.getBytes().length);
-        zos.putNextEntry(entry);
-        zos.write(data.getBytes());
-        zos.closeEntry();
+
+        entries.put(filePath, xmlBytes);
       }
       // add Library Resources to Export
-      addLibraryResources(jsonParser, xmlParser, bundle, zos);
-
-      entry = new ZipEntry(exportFileName + ".html");
-      entry.setSize(humanReadableWithCSS.getBytes().length);
-      zos.putNextEntry(entry);
-      zos.write(humanReadableWithCSS.getBytes());
-      zos.closeEntry();
-    } catch (InternalServerException | IOException ex) {
-      // TODO Auto-generated catch block
-      ex.printStackTrace();
-    } finally {
-      try {
-        zos.close();
-      } catch (IOException e) {
-        // TODO Auto-generated catch block
-        e.printStackTrace();
+      List<Library> libraries = getLibraryResources(bundle);
+      for (Library library1 : libraries) {
+        String json = jsonParser.setPrettyPrint(true).encodeResourceToString(library1);
+        String xml = xmlParser.setPrettyPrint(true).encodeResourceToString(library1);
+        String fileName = RESOURCES_DIRECTORY + library1.getName() + "-" + library1.getVersion();
+        entries.put(fileName + ".json", json.getBytes());
+        entries.put(fileName + ".xml", xml.getBytes());
       }
-    }
-  }
 
-  private void addLibraryResources(
-      IParser jsonParser,
-      IParser xmlParser,
-      org.hl7.fhir.r4.model.Bundle bundle,
-      ZipOutputStream zos)
-      throws IOException {
-    ZipEntry entry;
-    List<Library> libraries = getLibraryResources(bundle);
-    for (Library library : libraries) {
-      String json = jsonParser.setPrettyPrint(true).encodeResourceToString(library);
-      String xml = xmlParser.setPrettyPrint(true).encodeResourceToString(library);
-
-      String fileName = RESOURCES_DIRECTORY + library.getName() + "-" + library.getVersion();
-      entry = new ZipEntry(fileName + ".json");
-      entry.setSize(json.getBytes().length);
-      zos.putNextEntry(entry);
-      zos.write(json.getBytes());
-      zos.closeEntry();
-      entry = new ZipEntry(fileName + ".xml");
-      entry.setSize(xml.getBytes().length);
-      zos.putNextEntry(entry);
-      zos.write(xml.getBytes());
-      zos.closeEntry();
+      entries.put(exportFileName + ".html", humanReadableWithCSS.getBytes());
+    } catch (InternalServerException ex) {
```

Comment:
could this be removed(if not required)?

Reply:
This is still necessary

It's creating a map<string,byte[]> of "filename"->data and that is being sent to the ZipUtility to get packaged.

Reply:
Ah.. you were talking about the catch.. because the exception isn't being thrown anymore.   Removing..